### ISSUE TITLE
Access Denied

### ISSUE THREAD

**Issue by Nikunj1221** [pbatard/rufus#2390]
﻿

Checklist
---------
- [x] I looked at https://github.com/pbatard/rufus/wiki/FAQ to see if my question has already been answered.
- [x] I performed a search in the issue tracker for similar issues using keywords relevant to my problem, such as the error message I got from the log.
- [x] I clicked the 'Log' button (🗒️) or pressed <kbd>Ctrl</kbd>-<kbd>L</kbd> in Rufus, or used [DebugView](https://learn.microsoft.com/en-us/sysinternals/downloads/debugview), and copy/pasted the log into the section that says `<FULL LOG>` below.
- [x] The log I am copying is the FULL log, starting with the line `Rufus version: x.y.z` - I have NOT removed any part of it.

Additionally (if applicable):
- [ ] I ran a bad blocks check, by clicking _Show advanced format options_ then _Check device for bad blocks_, and confirmed that my USB is not defective.
- [x] I also tried one or more of the following:
  - [x] Using a different USB drive.
  - [x] Plugging the USB into a different port.
  - [x] Running Rufus on a different computer.
- [ ] If using an image, I clicked on the `(✓)` button to compute the MD5, SHA1 and SHA256 checksums, which are therefore present in the log I copied. I confirmed, by performing an internet search, that these values match the ones from the official image.

Issue description
-----------------

[0x00000005] Access is denied.
A device was eliminated because it didn't report itself as a disk
I am using an External SSD which I am able to access, but in RUFUS it is not detected as a disk

Log
---
```
Rufus x64 v4.3.2090 (Portable)
Windows version: Windows 11 Home Single Language x64 (Build 22621.2861)
Syslinux versions: 4.07/2013-07-25, 6.04/pre1
Grub versions: 0.4.6a, 2.06
System locale ID: 0x0409 (en-US)
Will use default UI locale 0x0409
SetLGP: Successfully set NoDriveTypeAutorun policy to 0x0000009E
Localization set to 'en-US'
Notice: The ISO download feature has been deactivated because 'Check for updates' is disabled in your settings.
Found 517 officially revoked UEFI bootloaders from embedded list
Found 2351 additional revoked UEFI bootloaders from this system's SKUSiPolicy.p7b
Processing Hub 1:
  Hub[1] = '\\?\usb#root_hub30#5&37dfea2b&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8}'
  Found ID[120]: USB\VID_0408&PID_5365\6&F567541&0&3
Processing Hub 2:
  Hub[2] = '\\?\usb#root_hub30#5&14de47ce&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8}'
  Found ID[129]: USB\VID_0BDA&PID_2852\00E04C000001
  Found ID[103]: USB\VID_0BDA&PID_9210\MSFT30012345679039
Processing IDs belonging to 'UASPSTOR':
  USB\VID_0BDA&PID_9210\MSFT30012345679039
Processing 'SCSI' device:
  Hardware ID: ''
Found non-USB non-removable device 'WDC PC SN530 SDBPNPZ-512G-1006' => Eliminated
Processing 'SCSI' device:
  Hardware ID: 'SCSI\DiskRealtek_RTL9210B-CG_____1.00'
  Matched with ID[103]: USB\VID_0BDA&PID_9210\MSFT30012345679039
  Matched with Hub[2]: '\\?\usb#root_hub30#5&14de47ce&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8}'
Found UAS (USB 3.0) device 'Realtek RTL9210B-CG UAS Device' (0BDA:9210)
Could not open '\\?\scsi#disk&ven_realtek&prod_rtl9210b-cg#7&560aeb3&0&000000#{53f56307-b6bf-11d0-94f2-00a0c91efb8b}': [0x00000005] Access is denied.
A device was eliminated because it didn't report itself as a disk
0 devices found

```

**Comment by pbatard** [comment]
https://github.com/pbatard/rufus/wiki/FAQ#error-0x00000005-access-is-denied

Something external to Rufus is preventing Rufus from accessing the drive.

I'm afraid that there is nothing I can do about it (and no, the fact that File Explorer is able to access it does not mean that Rufus should be able to access it, because Rufus needs to perform operations that File Explorer doesn't and relies on devices to **properly** answer additional I/O requests).

In short `Access Denied` means what it means, which is that, on your computer and with the hardware you are using, something is actively denying access to the device. You will therefore have to find what it is, as Rufus can not do magic here.
